buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidLibTarget.groovy

```diff
@@ -22,4 +26,23 @@ class AndroidLibTarget extends AndroidTarget {
     ManifestMerger2.MergeType getMergeType() {
         return ManifestMerger2.MergeType.LIBRARY
     }
+
+    List<String> getKotlincArguments() {
+        ImmutableList.Builder<String> extraKotlincArgs = ImmutableList.<String>builder()
+        if (hasKotlinAndroidExtensions) {
+            String options = ""
+            getResDirs().each {  String dir ->
+                // :root:module -> /root/module/
+                String module = project.path.replace(":", "/") + "/"
```

Comment:
Use `System.PathSeparator` instead of `/`

Reply:
👍

Reply:
This is still not addressed